crypto/fipsmodule/ec/asm/p256-armv8-asm.pl

```diff
@@ -166,30 +166,6 @@
 	ret
 .size	ecp_nistz256_sqr_mont,.-ecp_nistz256_sqr_mont
 
-// void	ecp_nistz256_add(BN_ULONG x0[4],const BN_ULONG x1[4],
```

Comment:
Forgot to mention in the first review, isn't it a bit weird that before ( before deleting this function) the compiler didn't complain that there are two functions with the same name (this one and the one in p256-nistz.c)?

Reply:
Agreed, this is weird. I guess the compiler may select the method (static defined in p256-nistz.c) before searching other alternatives. I think we may need a separate SIM to track down.

Reply:
The reason we removed this function is `delocate.go` does not allow duplicate symbol -- `ecp_nistz256_add`.